src/main/java/cc/hyperium/mods/statistics/GeneralStatisticsTracking.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Hyperium Client, Free client with huds and popular mod
+ * Copyright (C) 2018  Hyperium Dev Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published
+ *  by the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package cc.hyperium.mods.statistics;
+
+import cc.hyperium.config.ConfigOpt;
+import cc.hyperium.event.ChatEvent;
+import cc.hyperium.event.InvokeEvent;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class GeneralStatisticsTracking {
```

Comment:
If this was moved to a different package, why not delete the old one? I prefer where it is currently in mods/statistics tbh